README.md

```diff
@@ -261,9 +261,9 @@ less flexibility:
 
 ## Runtime Caching
 
-It's often desireable, even necessary to use precaching and runtime caching together. You may have seen our [sw-toolbox](https://github.com/GoogleChrome/sw-toolbox) tool, which handles runtime caching, and wondered how to use them together. Fortunately, sw-precache handles this fro you.
+It's often desireable, even necessary to use precaching and runtime caching together. You may have seen our `[sw-toolbox](https://github.com/GoogleChrome/sw-toolbox)` tool, which handles runtime caching, and wondered how to use them together. Fortunately, `sw-precache` handles this fro you.
```

Comment:
`[sw-toolbox](https://github.com/GoogleChrome/sw-toolbox)`

needs to be

    [`sw-toolbox`](https://github.com/GoogleChrome/sw-toolbox)